portality/models/oaipmh.py

```diff
@@ -71,8 +72,9 @@ def list_records(self, from_date=None, until_date=None, oai_set=None, list_size=
         if start_after is not None or from_date is not None or until_date is not None or oai_set is not None:
 
             if oai_set is not None:
+                a = oai_set.replace(constants.SUBJECTS_SCHEMA,"")
```

Comment:
variable `a` never used. I think is forgot to remove this line.

Reply:
@amdomanska can you confirm if this line should be used somewhere?

Reply:
This was a bug, that line is required, but the result of it was not used.  I have fixed and pushed.